### PR TITLE
[scarb-doc] Fix `trait_item_names`

### PR DESCRIPTION
**Pull request by drknzz**
- use cairo at [290f51f554c978180ca9d91177423305959cbed5](https://github.com/starkware-libs/cairo/commit/290f51f554c978180ca9d91177423305959cbed5)
- trait_item_names -> trait_required_item_names

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -119,8 +119,8 @@ toml = "0.8"
 toml_edit = { version = "0.22", features = ["serde"] }
 tower-http = { version = "0.4", features = ["fs"] }
 tracing = "0.1"
-tracing-log = "0.2"
 tracing-core = "0.1"
+tracing-log = "0.2"
```
Comment (maciektr) [inline Cargo.toml:123]: why?

Comment (drknzz) [reply Cargo.toml:123]: It just got moved one line below D:

Comment (drknzz) [reply Cargo.toml:123]: I've used this
```
cargo xtask set-cairo-version --rev 290f51f554c978180ca9d91177423305959cbed5
```

Comment (drknzz) [reply Cargo.toml:123]: It sorts the deps from what I see
https://github.com/software-mansion/scarb/blob/110826a7abf9b61d6dcee7da38a1538fb1298470/xtask/src/set_cairo_version.rs#L79

**Review by maciektr — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (maciektr) [reply Cargo.toml:123]: Cool, we must have missed it then!

**Pull request merged**